src/fenics_constitutive/stress_strain.py

```diff
@@ -132,9 +138,198 @@ def strain_from_grad_u(grad_u: np.ndarray, constraint: Constraint) -> np.ndarray
             strain_view[:, 1] = grad_u_view[:, 4]
             strain_view[:, 2] = grad_u_view[:, 8]
             strain_view[:, 3] = 1 / 2**0.5 * (grad_u_view[:, 1] + grad_u_view[:, 3])
-            strain_view[:, 4] = 1 / 2**0.5 * (grad_u_view[:, 5] + grad_u_view[:, 7])
-            strain_view[:, 5] = 1 / 2**0.5 * (grad_u_view[:, 2] + grad_u_view[:, 6])
+            strain_view[:, 4] = 1 / 2**0.5 * (grad_u_view[:, 2] + grad_u_view[:, 6])
+            strain_view[:, 5] = 1 / 2**0.5 * (grad_u_view[:, 5] + grad_u_view[:, 7])
```

Comment:
This just switches the order of the shear components. Our goal was to be compatible with Abaqus implicit which has the order 

$$
[\sigma_{11}, \sigma_{22}, \sigma_{33},\sigma_{12},\sigma_{13},\sigma_{23}]
$$

But I used this order by accident:

$$
[\sigma_{11}, \sigma_{22}, \sigma_{33},\sigma_{12},\sigma_{23},\sigma_{13}]
$$

For the models in this repository, this had no adverse effects because everything up to this point has been isotropic. But I had issues in the paper with an Abaqus UMAT which is not isotropic.